test-utils/test-utils-core/src/main/java/uk/gov/justice/services/test/utils/core/http/HttpResponsePoller.java

```diff
@@ -20,6 +20,10 @@
 import javax.json.JsonReader;
 import javax.ws.rs.core.Response;
 
+/**
+ *
+ */
+@Deprecated
```

Comment:
Do we want to deprecate at the moment?  If we do then this needs a comment of where to go for the new version.